packages/mobile/src/exchange/CeloGoldHistoryChart.tsx

```diff
@@ -0,0 +1,295 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import _ from 'lodash'
+import React, { useCallback, useState } from 'react'
+import { WithTranslation } from 'react-i18next'
+import { ActivityIndicator, LayoutChangeEvent, StyleSheet, Text, View } from 'react-native'
+import { Circle, G, Line, Text as SvgText } from 'react-native-svg'
+import { useExchangeRate } from 'src/exchange/hooks'
+import { exchangeHistorySelector } from 'src/exchange/reducer'
+import { Namespaces, withTranslation } from 'src/i18n'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { convertDollarsToLocalAmount } from 'src/localCurrency/convert'
+import { useLocalCurrencyCode } from 'src/localCurrency/hooks'
+import { getLocalCurrencyExchangeRate } from 'src/localCurrency/selectors'
+import useSelector from 'src/redux/useSelector'
+import { goldToDollarAmount } from 'src/utils/currencyExchange'
+import { getLocalCurrencyDisplayValue } from 'src/utils/formatting'
+// @ts-ignore
```

Comment:
Why the ts-ignore?

Reply:
Cause victory charts library does not have type binding and next line causing a type error.

Reply:
https://github.com/FormidableLabs/victory-native/issues/269